### ISSUE TITLE
fix link in header

### ISSUE THREAD

**Issue by mmaelicke** [MuellerSeb/GSTools-EGU2020#1]
Hey,
Awesome, to see you using notebook-classroom! Thanks!

Just a quick note: There is a config option in configuration.js to make the Brand link work, it's called `homeUrl`. you can place e.g.:
````javascript
    url: "https://muellerseb.github.io/GSTools-EGU2020",
    "homeUrl": "https://geostat-framework.readthedocs.io/projects/gstools/",
     navigation: [ ... ]
```
And the 'GSTools EGU2020 Presentation' will point to your docs.
See you @ EGU

**Comment by MuellerSeb** [comment]
I couldn't resist using this awesome app ;-)
And thanks for the hint! I also needed to update the `workflows/main.yml` to make it work (generation of static files).

**Comment by mmaelicke** [comment]
Yeah, I'll have to admit that the https://github.com/mmaelicke/sample-lecture is a bit outdated. I will replace it with https://github.com/hydrocode-de/sample-lecture-python and https://github.com/hydrocode-de/sample-lecture-r
